db/migrate/20210310154720_add_description2_to_charity.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescription2ToCharity < ActiveRecord::Migration[6.0]
+  def change
+    add_column :charities, :description2, :text
```

Comment:
Is it possible to only use 1 description field?

Reply:
sure will remove this thanks!